googletest/plan.sh

```diff
@@ -44,7 +42,7 @@ do_prepare() {
 
   # patch version as it is suppose to be 1.8.1
   # ref: https://github.com/google/googletest/issues/1816
-  sed -e "s/GOOGLETEST_VERSION 1.9.0/GOOGLETEST_VERSION ${pkg_version}/" -i CMakeLists.txt
+  # sed -e "s/GOOGLETEST_VERSION 1.9.0/GOOGLETEST_VERSION ${pkg_version}/" -i CMakeLists.txt
```

Comment:
Don’t need this anymore so we should remove it.

Reply:
Oops.

Reply:
Fixed